src/Settings/RunawayFluid.cpp

```diff
@@ -50,7 +50,7 @@ RunawayFluid *SimulationGenerator::ConstructRunawayFluid(FVM::Grid *g,
         Eceff_mode, ava_mode, compton_mode, compton_photon_flux
     );
 
-    delete cq;
+//    delete cq;
```

Comment:
Is this correct? If it is, maybe just remove?

Reply:
right, this is not completely finished. In the RunawayFluid constructor we create a new collqty settings objects that is to be used in all calculations. I want to create that here in the settings file instead, and generate nuS and nuD using that (previously we apparently got a valgrind error because cq (provided to nuS and nuD) was deleted, which was scary)